containers/withQueryLoader.tsx

```diff
@@ -23,8 +24,9 @@ const withQueryLoader = <T extends {}>(
   if (data) {
     return <Component queryData={data} {...props} />
   }
-
-  return <h1>No Data...</h1>
+  return (
```

Comment:
If it's no data that means the user is not signed in. We should redirect to sign in form.

Reply:
It redirects to login page now.